### PR TITLE
Include health status in tasks

### PR DESCRIPTION
**Pull request by drewrobb**
@mlerner

First work in golang!

### CODE REVIEW

**Comment by mlerner** [comment]
:boom:

File: services/marathon/marathon.go [hunk 1]
```diff
@@ -206,10 +217,20 @@ func createApps(tasksById map[string][]marathonTask, marathonApps map[string]mar
 		tasks := []Task{}
 		for _, mTask := range tasksById[appId] {
 			if len(mTask.Ports) > 0 {
+				// If task is within grace period, it has no
+				// healthCheckResults but startedAt is nil, otherwise
+				// startedAt is a timestamp string
+				var healthy = (len(mTask.StartedAt) > 4)
```
Comment (activars) [inline services/marathon/marathon.go]: For safety:
```
(mTask.StartedAt != nil && mTask.StartedAt > 4)
```

**Comment by activars** [comment]
I like the :boom:  reaction. haha. 
There's a minor issue, otherwise looks good.

File: services/marathon/marathon.go (hunk 1 above)
Comment (apriendeau) [inline services/marathon/marathon.go]: Just for reference, you can omit the wrapping parenthesis. Not a critique just a random fact for @drewrobb

**Pull request closed without merging**